### ISSUE TITLE
Stutter and crashing.

### ISSUE THREAD

**Issue by PixelMirror** [meepdarknessmeep/gproxy#12]
So, for some reason the game seems to start to stutter over time and eventually crashes from running out of memory. This seems to be related to hooks and get better or worse depending on what you're running in the hook but the final outcome always seem to be the same; stutter and an eventual crash.

**Comment by NERO2k** [comment]
It's the thing I was talking about. The code is not being run in the client state so a lot of functions are missing. I'm looking for a way to do it.

**Comment by PixelMirror** [comment]
Executing code in the client state would ruin the principle of it though.

**Comment by NERO2k** [comment]
That's true but I don't see any other way of doing it since the menu state cant interact with any other state anyways so there are several disadvantages.

**Comment by meepdarknessmeep** [comment]
There is, and it's interacting with the LuaJIT objects itself which is my project.

**Comment by meepdarknessmeep** [comment]
Which version are you running?

**Comment by NERO2k** [comment]
Myself I'm running the latest version.

**Comment by PixelMirror** [comment]
The version I'm having issues with crashing and stuttering is also the most recent version.

**Comment by NERO2k** [comment]
Any updates?

**Comment by meepdarknessmeep** [comment]
It's probably something with the override of GC.

**Comment by NERO2k** [comment]
Any chance we will see a fix for these 2 issues soon? I think they are related since if I try to use some client state functions my game crashes instantly. For example, i cant even get localplayer without an error.

**Comment by PixelMirror** [comment]
Any update on this?
